_posts/2025-9-20-Poetry.md

```
@@ -0,0 +1,638 @@
+poetry new --src my_project --name my_package
+```
+
+**Key Options:**
+- `--src`: Creates a `src/` directory layout (recommended for packaging)
+- `--name my_package`: Names the package differently from the project directory
+
+**Generated structure:**
+```
+my_project/
+├── pyproject.toml
+├── README.rst
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)